packages/browser-wallet-api/src/wallet-api.ts

```diff
@@ -92,6 +104,10 @@ class WalletApi extends EventEmitter implements IWalletApi {
     private handleEvent(type: EventType) {
         this.messageHandler.handleMessage(createEventTypeFilter(type), (msg) => this.emit(type, msg.payload));
     }
+
+    public getJsonRpcClient(): JsonRpcClient {
```

Comment:
Is there access restriction on this? A non-whitelisted website should not be able to use it.